### ISSUE TITLE
Files not properly processed: At submission the file never makes it to the editorial board

### ISSUE THREAD

**Issue by ryanfb** [papyri/sosol#62]
See: #56 #60 

Some of these may have been manually resubmitted per #56, so we need to check for that before unwedging/resubmitting them. Typically we need to check the branch, check that there's not a conflicting board branch, change identifier status to editing, change publication status to editing, check for submission reason, then resubmit.

See also: sosol/sosol#73

- [x] 4617
- [x] 4626
- [x] 4628
- [x] 7745
- [x] 8241
- [x] 8644
- [x] 8831
- [x] 8845
- [x] 9087
- [x] 9097
- [x] 9338
- [x] 9468
- [x] 9488
- [x] 9556
- [x] 10081
- [x] 10472
- [x] 10757
- [x] 13699
- [x] 13884
- [x] 14128
- [x] 14133
- [x] 14152
- [x] 14814
- [x] 14833
- [x] 15140
- [x] 15222
- [x] 15436
- [x] 15967
- [x] 15977
- [x] 16235
- [x] 16249
- [x] 16364
- [x] 16567
- [x] 16580
- [x] 16582
- [x] 16663
- [x] 16738
- [x] 16755
- [x] 16839
- [x] 16846
- [x] 16909
- [x] 17111
- [x] 17120
- [x] 17122
- [x] 17127
- [x] 17129
- [x] 17150
- [x] 17153
- [x] 17161
- [x] 17163
- [x] 17184
- [x] 17186
- [x] 17188
- [x] 17190
- [x] 17193
- [x] 17200
- [x] 17288
- [x] 17295
- [x] 17318
- [x] 17458
- [x] 17574
- [x] 17576
- [x] 17582
- [x] 17596
- [x] 17598
- [x] 17717
- [x] 17719
- [x] 17729
- [x] 18041
- [x] 18046
- [x] 18182
- [x] 19019
- [x] 19169
- [x] 19499
- [x] 19501
- [x] 19562
- [x] 19569
- [x] 19656
- [x] 19692
- [x] 19863
- [x] 20331
- [x] 20343
- [x] 20427
- [x] 20509
- [x] 20631
- [x] 22080
- [x] 22093
- [x] 22201
- [x] 22745
- [x] 22763
- [x] 22898
- [x] 22994
- [x] 23176
- [x] 23982
- [x] 24319
- [x] 24338
- [x] 24432
- [x] 24454
- [x] 24455
- [x] 24501
- [x] 25292
- [x] 25825
- [x] 25844
- [x] 25856
- [x] 26025
- [x] 26142
- [x] 26964
- [x] 27042
- [x] 27232
- [x] 27234
- [x] 27327
- [x] 27418
- [x] 27744
- [x] 28056
- [x] 28210
- [x] 28213
- [x] 28812
- [x] 28813
- [x] 29148
- [x] 29150
- [x] 29300
- [x] 29344
- [x] 29512
- [x] 29597
- [x] 29610
- [x] 29612
- [x] 29808
- [x] 30174
- [x] 30307
- [x] 30318
- [x] 30329
- [x] 30564
- [x] 30565
- [x] 30677
- [x] 30704
- [x] 30710
- [x] 30711
- [x] 30784
- [x] 31011
- [x] 31013
- [x] 31063
- [x] 32412
- [x] 32569
- [x] 32957
- [x] 33009
- [x] 33245
- [x] 33279
- [x] 33638
- [x] 33651
- [x] 33777
- [x] 33801
- [x] 33803
- [x] 35220
- [x] 36035
- [x] 37043
- [x] 37608
- [x] 37636
- [x] 37660
- [x] 37699
- [x] 38051
- [x] 38097
- [x] 38718
- [x] 38722
- [x] 38988
- [x] 39546
- [x] 42281
- [x] 42321
- [x] 43370
- [x] 44517
- [x] 44656

**Comment by jcowey** [comment]
Fantastic that you have found a way to identify these files. I knew of a shitload of them from the user:
http://papyri.info/editor/users/Georg

**Comment by ryanfb** [comment]
A routine to help unwedge these: https://gist.github.com/ryanfb/d37627b1eb9cf7b4ab3558532136430c

All publications here submitted by 'Catherine ..........' are caused/blocked by sosol/sosol#194

42321 has an unknown APIS identifier so it just (still) falls through the boards on resubmit.

**Comment by ryanfb** [comment]
13884 & 14133 have invalid branches on the owner repository.

Most of the 'Catherine ..........' publications have been unwedged by fixing sosol/sosol#194, with the exception of 37636 which runs into sosol/gitwrapper#4 but can't fall back into the git command (because of sosol/sosol#200).

**Comment by jcowey** [comment]
13884 and 14133 and 42321  are the remaining wedged files

**Comment by jcowey** [comment]
papyri.info/editor/publications/13884 (submitted 25.07.2011) concerned o.abu.mina;;460

http://www.papyri.info/ddbdp/o.abu.mina;;460
which was submitted two years later and is now fine. 

So this is no longer a problem

**Comment by jcowey** [comment]
papyri.info/editor/publications/14133 (submitted 27.07.2011) concerned bgu;19;2835

User is papyri.info/editor/users/Zoom

This should be retrieved and resubmitted

**Comment by ryanfb** [comment]
Looking into these now. 42321 belongs to Philip J. Katz, concerning [P.Amh. 2, 186](http://papyri.info/hgv/39337) has the following as an APIS identifier (as `unknown.apis.2014-0001`, which is what causes it to fall through the boards on submit):
```
<?xml version="1.0" encoding="UTF-8"?><TEI xmlns="http://www.tei-c.org/ns/1.0">
   <teiHeader>
      <fileDesc>
         <titleStmt>
           <title>Tax Receipt</title>
         </titleStmt>
         <publicationStmt>
            <authority>APIS</authority>
           <idno type="apisid">morgan.apis.1</idno>
           <idno type="TM">39337</idno>
         </publicationStmt>
         <sourceDesc>
            <msDesc>
               <msIdentifier>
                 <idno type="invNo">Amh. Gr. Pap. 186</idno>
               </msIdentifier>
               <msContents>
                 <summary>Tax-receipt written in a small hand with numerous abbreviations. Seventh or eighth century A.D. Complete. 6 lines.</summary>
                 <msItemStruct>
                   <textLang mainLang="grc">Greek</textLang>
                 </msItemStruct>
               </msContents>
               <physDesc>
                  <objectDesc>
                     <supportDesc>
                       <support>9.5 x 7.9 cm</support>
                       <condition>
                        </condition>
                     </supportDesc>
                     <layoutDesc>
                        <layout>
                        </layout>
                     </layoutDesc>
                  </objectDesc>
               </physDesc>
               <history>
                  <origin>
                    <origDate notBefore="0600" notAfter="0799">VII or VIII</origDate>
                    <origPlace>unknown</origPlace>
                  </origin>
                  <provenance>
                    <p>Bib ID 286896. Morgan Library Accession Number Amh. Gr. Pap. 186. Lord Amherst Tyssen-Amherst, 1st Baron Amherst of Hackney; purchased by J. Pierpont Morgan (1837-1913) in 1912; J.P. Morgan (1867-1943).</p>
                  </provenance>
               </history>
            </msDesc>
         </sourceDesc>
      </fileDesc>
      <profileDesc>
         <langUsage>
            <language ident="en">English</language>
            <language ident="la">Latin</language>
            <language ident="el">Greek</language>
         </langUsage>
         <textClass>
            <keywords scheme="#apis">
              <term>Taxes</term>
              <term>Receipt</term>
              <term>
                <rs type="genre_form">Documentary</rs>
              </term>
              <term>
                <rs type="genre_form">Papyrus</rs>
              </term>
            </keywords>
         </textClass>
      </profileDesc>
      <revisionDesc>
        <change when="2014-12-22T11:47:09-05:00" who="http://papyri.info/editor">Automated creation from template</change>
      </revisionDesc>
   </teiHeader>
   <text>
      <body>
         <div type="bibliography" subtype="citations">
            <listBibl>
              <bibl>citation</bibl>
            </listBibl>
         </div>
      </body>
   </text>
</TEI>
```
No changes have been made to the HGV record, and we don't have an APIS board/folder for the Morgan: https://github.com/papyri/idp.data/tree/master/APIS

Any ideas on what we should do with this publication?

**Comment by ryanfb** [comment]
> http://papyri.info/editor/publications/14133 (submitted 27.07.2011) concerned bgu;19;2835
> 
> User is http://papyri.info/editor/users/Zoom
> 
> This should be retrieved and resubmitted

This publication has the branch name `committed/2011/07/27/BGU_19,_2835`, which doesn't exist in the `repo/users/Zoom.git` repository. It does exist in `repo.bak/users/Zoom.git` under `packed-refs` as `2e3cb641c63d255af0b972f14c6da4a9c1ae7ec0`. There's no loose object in any repo corresponding to that hash, and checking all the pack files with `git verify-pack -v` none of them have it either. So probably at some point the pack containing it was corrupted and eventually removed. I'm currently trying to see if it's possible find a dangling parent commit (in the backup repo) with `git fsck --unreachable`, but it's possible that this publication is unrecoverable.

**Comment by ryanfb** [comment]
After a fair amount of Git archaeology, I don't know that 14133 = BGU 19 2835 is recoverable. As atonement for this problem, here's BGU 19 2835:
![scanbot jan 2 2018 9 39 am](https://user-images.githubusercontent.com/19781/34492055-f7dbc2dc-efb3-11e7-9ccd-d6e1ba652018.png)
![untitledpg2](https://user-images.githubusercontent.com/19781/34492059-fd13ccf4-efb3-11e7-9214-cdff9ed778c9.png)
And my partial Leiden+ transcription (I wasn't sure how to interpret `/` or `//`, and I'm sure I've gotten some other things wrong:
```
<S=.grc
<=
1. lost.?lin[
1. ].1[.5]ρ̣[
2. τ]ῶν λαμπροτάτω̣[ν
3. ]μ̣.2ω̣ι̣ τ̣.3[.2] απ̣.4[
4. Ἑρμοῦ πόλεως] τῆς λαμπροτάτης .1[.4].2[
5. π]αρὰ Αὐρηλίου Κοπρέου Σ̣[.4].1[
6. μητρὸς] Σοήρεως καὶ Παταυ̣ρι.3[
7. ].2 τῶν αὐτῶν γονέω̣[ν] ἀ̣π̣ὸ .2[
8. Π]α̣υσίρεως μητρὸς Εἰρή̣[νη]ς̣ κ̣[α]ὶ̣ [
9. ]ν Πεμοῦτο̣ς̣ καὶ Πεγευ.6[
10. ἀπ]ὸ ἐποικίου Βερ[κ]ὺ τοῦ Ἑρμο[υπ]ο[λείτου
11. ].1 χαίρειν· ὁμο̣λ̣ογοῦμε̣[ν
12. ἐσχηκέναι κα]ὶ μεμετρῆσθαι παρά σου πυρ̣ο̣[ῦ
13. ].1ς β εἰς κατασπορὰν τῆ[ς
14. εὐτυχο]ῦς γ ἤτ[οι] ι̣η (ἰνδικ(τίονος)) .2φ.1[
15. σ]ὺ̣ν διαφόρ̣[ῳ ἡμι]ολίῳ ἀρτάβα̣ς̣ [
16. ].1 τα ἡμισ̣[.2].3[
17. ].2[.9].1λ̣.1[
17. ]lost.?lin
=>
```
Hopefully this is enough that we can create a new publication off of papyri.info/hgv/91745 and resubmit it, giving credit to the original submitter and explaining the problem.

**Comment by jcowey** [comment]
Many thanks for this. It will be re-entered. Was that you using your 

https://dcthree.github.io/antigrapheus/

tool?

**Comment by ryanfb** [comment]
All hand-typed - I figured it would be easier than catching OCR mistakes...

**Comment by jcowey** [comment]
Well then an even bigger thank you. What a way to be starting 2018 ! ;)

**Comment by ryanfb** [comment]
Moved 42321 into its own issue.
